### DISCUSSION TITLE
How to disable motion thumbnail on top of screen when in Camera Live View

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#12049] [General]
In View I only want to see the "Live Cameras" not event thumbnails (with motion) on top of page. I can't seem to find a setting to disable it the thumbs. Is it even possible?

Version .14 Beta 3

**Comment by NickM-27** [comment]
this is not possible currently, maybe in a future version

**Reply by 18923** [reply]
Thanks for the prompt response.

**Reply by 18923** [reply]
And thank you for the option to only show a static image. I find the looping videos distracting.

_**Play Alert Videos**
By default, recent alerts on the Live dashboard play as small looping videos. Disable this option to only show a static image of recent alerts on this device/browser._

Frigate is amazing...
